### DISCUSSION TITLE
图目录的第二页行与行之间莫名其妙的出现大量空白

### DISCUSSION THREAD

**Discussion by Funys0** [TheNetAdmin/zjuthesis#297] [General]
在修改文章后，添加了几个图片，编译后发现图目录第二页行与行之间莫名其妙的出现了很大的空白，但是到了第三页又恢复正常了。请教大家，这是啥情况，第一次遇到。
![Uploading 2023-01-12_18-55.png…]()

**Comment by DarrenLord** [comment]
就是页面正常排版，图片所需位置和后面的格式需要的行距加在一起不足以在这一面一起展示，它就会自动将后面的在下一面展示。当然也会有同一面，两段二级/三级文字之间有大量空白，这是因为后面的图片排版所影响的。
在word上也会有这种情况出现，

**Reply by Funys0** [reply]
只是论文开始时的图目录，这个应该是一条一条的图片标题跟位置信息，只有单纯的文字为啥还会有排版的问题

**Comment by mitmonk** [comment]
遇到了一样问题，不知道楼主有没有解决。主要是图目录第一页空间没有用完就换页了，留了半页的空白

**Reply by Funys0** [reply]
推荐在toc.tex中插入\fontsize{12pt}{12.5pt}\selectfont，里面那个12.5pt是我试出来的值(可以自己调节)，不细看的话看不太出来行间距变了。
![image](https://user-images.githubusercontent.com/122529122/232119847-2be60a53-a530-4980-bc24-0bd791064763.png)

**Reply by mitmonk** [reply]
真实有效，太强了

**Comment by mitmonk** [comment]
遇到了一样问题，不知道楼主有没有解决。主要是图目录第一页空间没有用完就换页了，留了半页的空白

**Reply by Funys0** [reply]
\listoffigures生成的东西莫名其妙的出现大行间距，上面有作者的解释但是并不能解决问题。
这东西貌似是通病，其它学院给的模板也有类似的问题，应该已经好久了但是所有作者没有解决。
